src/elements/dialog/dialog/dialog.stories.ts

```diff
@@ -286,7 +286,7 @@ const StepperTemplate = ({
   <sbb-dialog trigger="dialog-trigger" ${sbbSpread(args)}>
     ${dialogTitle(level)}
     ${includeCloseButton ? html`<sbb-dialog-close-button></sbb-dialog-close-button>` : nothing}
-    <sbb-dialog-content>
+    <sbb-dialog-content style="padding-block-start: 1rem">
```

Comment:
Why is this necessary? I worry about additional style attributes in stories, as they might be copied without understanding.